### PR TITLE
Bump version to v0.3.6

### PR DESCRIPTION
**Pull request by dhruvmanila**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
 # Changelog
 
+## 0.3.6
+
+### Preview features
+
+- \[`pylint`\] Implement `bad-staticmethod-argument` (`PLW0211`) ([#10781](https://github.com/astral-sh/ruff/pull/10781))
+- \[`pylint`\] Implement `if-stmt-min-max` (`PLR1730`, `PLR1731`) ([#10002](https://github.com/astral-sh/ruff/pull/10002))
+- \[`pyupgrade`\] Replace str,Enum with StrEnum `UP042` ([#10713](https://github.com/astral-sh/ruff/pull/10713))
```
Comment (AlexWaygood) [inline CHANGELOG.md:9]:
```suggestion
- \[`pyupgrade`\] Replace `str,Enum` multiple inheritance with `StrEnum` `UP042` ([#10713](https://github.com/astral-sh/ruff/pull/10713))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+- \[`refurb`\] Implement `if-expr-instead-of-or-operator` (`FURB110`) ([#10687](https://github.com/astral-sh/ruff/pull/10687))
+- \[`refurb`\] Implement `int-on-sliced-str` (`FURB166`) ([#10650](https://github.com/astral-sh/ruff/pull/10650))
+- \[`refurb`\] Implement `write-whole-file` (`FURB103`) ([#10802](https://github.com/astral-sh/ruff/pull/10802))
+- \[`refurb`\] Support `itemgetter` in `reimplemented-operator` (`FURB118`) ([#10526](https://github.com/astral-sh/ruff/pull/10526))
+- \[`flake8_comprehensions`\] Add `sum`/`min`/`max` to unnecessary comprehension check (`C419`) ([#10759](https://github.com/astral-sh/ruff/pull/10759))
+
+### Rule changes
+
+- \[`pydocstyle`\] Require capitalizing single word sentence (`D403`) ([#10776](https://github.com/astral-sh/ruff/pull/10776))
```
Comment (AlexWaygood) [inline CHANGELOG.md:18]:
```suggestion
- \[`pydocstyle`\] Require capitalizing docstrings where the first sentence is a single word (`D403`) ([#10776](https://github.com/astral-sh/ruff/pull/10776))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,59 @@
...
+- Respect `# noqa` directives on `__all__` openers ([#10798](https://github.com/astral-sh/ruff/pull/10798))
+
+### Documentation
+
+- Correct some oversights in the documentation from #10756 ([#10796](https://github.com/astral-sh/ruff/pull/10796))
+- Rework docs for pydocstyle rules ([#10754](https://github.com/astral-sh/ruff/pull/10754))
+- \[`flake8_comprehensions`\] update docs for unnecessary-comprehension-any-all (C419) ([#10744](https://github.com/astral-sh/ruff/pull/10744))
+- Update `ruff_linter` crate name in CONTRIBUTING.md ([#10745](https://github.com/astral-sh/ruff/pull/10745))
+- More precise Pylint comparison ([#10756](https://github.com/astral-sh/ruff/pull/10756))
+- Change `Lint` to `Format` in formatter.md ([#10777](https://github.com/astral-sh/ruff/pull/10777))
```
Comment (charliermarsh) [inline CHANGELOG.md:55]: I think this section is fine to remove, personally.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,59 @@
...
+- \[`pydocstyle`\] Require capitalizing single word sentence (`D403`) ([#10776](https://github.com/astral-sh/ruff/pull/10776))
+- \[`pycodestyle`\] Ignore annotated lambdas in class scopes (`E731`) ([#10720](https://github.com/astral-sh/ruff/pull/10720))
+- \[`flake8-pyi`\] Various improvements to PYI034 ([#10807](https://github.com/astral-sh/ruff/pull/10807))
+- \[`flake8-slots`\] Flag subclasses of call-based `typing.NamedTuple`s as well as subclasses of `collections.namedtuple()` (`SLOT002`) ([#10808](https://github.com/astral-sh/ruff/pull/10808))
+- \[`pyflakes`\] Allow forward references in class bases in stub files (`F821`) ([#10779](https://github.com/astral-sh/ruff/pull/10779))
+- \[`pygrep-hooks`\] Improve `blanket-noqa` error message (`PGH004`) ([#10851](https://github.com/astral-sh/ruff/pull/10851))
+
+### CLI
+
+- Support FORCE_COLOR env var ([#10839](https://github.com/astral-sh/ruff/pull/10839))
```
Comment (AlexWaygood) [inline CHANGELOG.md:27]:
```suggestion
- Support `FORCE_COLOR` env var ([#10839](https://github.com/astral-sh/ruff/pull/10839))
```

**Review by charliermarsh — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (dhruvmanila) [reply CHANGELOG.md:55]: Do you mean the whole "Documentation" section?

Yeah, it seems the changes are small enough.

**Review by AlexWaygood — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (charliermarsh) [reply CHANGELOG.md:55]: Yeah, they're just not important for users that are looking to upgrade or trying to understand what's changed. I typically omit the whole "Documentation" section.

**Pull request merged**